tests/test_main_forge_handlers.py

```
@@ -0,0 +1,539 @@
+class TestCreateStart(unittest.TestCase):
+    """create_start initialises a ForgeDraft and returns WAITING_TITLE."""
+
+    def test_returns_waiting_title_via_message(self):
+        update = _make_callback_update.__wrapped__ if hasattr(_make_callback_update, "__wrapped__") else None
+        update = MagicMock()
+        update.callback_query = None
+        update.message = MagicMock()
+        update.message.reply_text = AsyncMock()
+        ctx = _make_context()
+        result = _run(create_start(update, ctx))
+        self.assertEqual(result, WAITING_TITLE)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)